components/dom_crawler.rst

```diff
@@ -338,31 +338,31 @@ This behavior is best illustrated with examples::
 
     $crawler->filterXPath('//span[contains(@id, "article-")]')->evaluate('substring-after(@id, "-")');
     /* array:3 [
-        0 => "100",
-        1 => "101",
-        2 => "102",
+      0 => "100"
```

Comment:
This reverts the changes made in the previous PR. Is that OK?

Reply:
Yes unless we found a decision. I opened a thread in the team forum. 👍🏻